prospector/tools/mypy/__init__.py

```diff
@@ -1,14 +1,21 @@
+import json
 from multiprocessing import Process, Queue
-from typing import TYPE_CHECKING, Any, Callable, Optional
-
-from mypy import api
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Optional,
+)
+
+import mypy.api
+import mypy.build
+import mypy.errors
+import mypy.fscache
+import mypy.main
 
 from prospector.finder import FileFinder
 from prospector.message import Location, Message
 from prospector.tools import ToolBase
-
-__all__ = ("MypyTool",)
-
```

Comment:
This break the API, why do it ?

Reply:
Oups, it's a mistake, thanks for the note. :-)